OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -383,12 +383,15 @@ public bool IsVisible(Actor a)
 
 		public bool IsTargetable(Actor a)
 		{
-			if (a.TraitsImplementing<IVisibilityModifier>().Any(t => !t.IsVisible(a, self.Owner)))
+			if (a.TraitsImplementing<IStealth>().Any(t => !t.IsVisible(a, self.Owner)))
```

Comment:
Conceptually speaking, cloaking, stealthing, etc. is a mod-specific mechanic with which Shroud should not have anything to do with.